README.md

```diff
@@ -1,50 +1,49 @@
 Campaign Central
 ================
 
- 
-Running
-=======
+Install and Setup
+=================
```

Comment:
Might be better to use '##' for level 2 heading level so that it's consistent with the level 3s.